scipy/stats/__init__.py

```diff
@@ -271,6 +271,7 @@
    combine_pvalues
    jarque_bera
    page_trend_test
+   quartile_coeff_dispersion
```

Comment:
Does this belong in statistical tests, or might summary statistics be more appropriate?

Reply:
Yea, I guess summary statistics is more appropriate.